cost-analyzer/templates/_helpers.tpl

```diff
@@ -31,6 +31,15 @@ Set important variables before starting main templates
   {{- end -}}
 {{- end -}}
 
+{{/*
```

Comment:
QQ here, should we go back and edit notes for 2.3.0? Thinking at least call out this same messaging, but that we don't fail on it in 2.3.0?

Reply:
yes, edit notes. I'll create a message for that and the Thena blast